porcupine/plugins/sidebar.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+import tkinter as tk
+import tkinter.ttk as ttk
+from typing import Dict, List
+
+from porcupine import get_paned_window, get_tab_manager, tabs
+
+
+class Sidebar(ttk.Treeview):
+
+    def __init__(self, master: tk.BaseWidget) -> None:
+        super().__init__(master)
```

Comment:
To prevent selecting multiple items at once:

```suggestion
        super().__init__(master, selectmode='browse')
```